agents/superpowers/specs/2026-07-11-plugin-blogger-design.md

```
@@ -0,0 +1,269 @@
+### `Post`
+
+```
+name:    string (title, label)
+summary: string (optional; shown on PostCard)
+outline: Ref<Markdown.Document>               // planning / ideas / notes
+drafts:  Schema.Array(Ref<Draft>)             // versions, order = version order
+```
+
+### `Draft` (wrapper object so versions carry metadata)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)